code/__HELPERS/html_assistant.dm

```diff
@@ -0,0 +1,2 @@
+/proc/get_wiki_url()
+	return CONFIG_GET(string/wikiurl)
```

Comment:
this seems a weird place to put this proc, like it would belong more next to the config definition itself maybe

Reply:
why is this even a proc

Reply:
it had indentation errors when I used GONFIG_GET before. I tried it again and now it is fine. hmm...